mlflow/pyspark/ml/__init__.py

```diff
@@ -39,13 +46,33 @@
 
 
 def _read_log_model_allowlist_from_file(allowlist_file):
-    allowlist = set()
-    with open(allowlist_file) as f:
-        for line in f:
+    def _parse_allowlist_file(file, allowlist):
```

Comment:
```suggestion
    def _parse_allowlist_file(file):
```

Can we remove this argument?